src/libData/AccountData/AccountStore.h

```diff
@@ -106,6 +106,8 @@ class AccountStore
 
   bool SerializeDelta();
 
+  AccountStoreTemp& GetAccountStoreTemp() { return *m_accountStoreTemp; }
```

Comment:
Add const at the end of function for get function

Reply:
This func is meant for changing the reference that it returns